LayoutTests/fast/forms/input-text-maxlength.html

```diff
@@ -25,10 +25,11 @@
     return document.getElementById(id).value;
 }
 function visibleValueOf(id) {
-    var el = document.getElementById(id);
-    el.focus();
-    document.execCommand('SelectAll');
-    return document.getSelection().toString();
+    const element = document.getElementById(id);
+    const shadowRoot = internals.shadowRoot(element);
```

Comment:
I noticed in other tests you avoided the need for this. But not practical here?

Reply:
Right, here, we specifically want to get the rendered text, not the value returned by input.value. That's separately tested by domValueOf function defined above.